torchvision/datasets/hmdb51.py

```diff
@@ -64,7 +65,17 @@ def __init__(self, root, annotation_path, frames_per_clip, step_between_clips=1,
         self.samples = make_dataset(self.root, class_to_idx, extensions, is_valid_file=None)
         self.classes = classes
         video_list = [x[0] for x in self.samples]
-        video_clips = VideoClips(video_list, frames_per_clip, step_between_clips)
+        video_clips = VideoClips(
+            video_list,
+            frames_per_clip,
+            step_between_clips,
+            frame_rate,
+            precomputed_metadata,
+            precomputed_metadata_filepath,
+        )
+        if save_metadata_filepath:
```

Comment:
I'm not sure if it should be up to the dataset to save the metadata, or the training code around it.
I'd say that this caching should be part of the training pipeline. Thoughts?

Reply:
@fmassa I agree with you. I will remove optional arguments `precomputed_metadata_filepath`, and `save_metadata_filepath`.